### ISSUE TITLE
2.0 ckpt files WONT LOAD on automatic1111

### ISSUE THREAD

**Issue by LIQUIDMIND111** [TheLastBen/fast-stable-diffusion#895]
i get error, even with the 768-v yaml installed correctly..... do i need to make a yaml file for custom models? IMPORTANT, i also noticed that 2.0 models LOAD using V prediction, but custom models, EVEN MADE WITH 2.0 will load on EPS prediction.... is there something i need to do so YOUR MODELS can be loaded on automatic 1111?

**Comment by LIQUIDMIND111** [comment]
OK FIXED, you need to make a YAML file for EACH MODEL.......

**Comment by TheLastBen** [comment]
you don't need to make the yaml files, just choose the right version in the dropdown menu in the last cell

**Comment by LIQUIDMIND111** [comment]
> you don't need to make the yaml files, just choose the right version in the dropdown menu in the last cell

ONLY ERRORS i get if i dont make a YAML file..... i tried restarting ALL, and nothing worked until i made a YAML file......

**Comment by LIQUIDMIND111** [comment]
they mention this on automatic wiki

grab [config](https://raw.githubusercontent.com/Stability-AI/stablediffusion/main/configs/stable-diffusion/v2-inference-v.yaml) from SD2.0 repository and put it into same place as the checkpoint, renaming it to have same filename (ie if your checkpoint is named 768-v-ema.ckpt, the config should be named 768-v-ema.yaml)
select the new checkpoint from the UI

**Comment by LIQUIDMIND111** [comment]
> you don't need to make the yaml files, just choose the right version in the dropdown menu in the last cell

seems THEY still have issues adapting it.

**Comment by LIQUIDMIND111** [comment]
> you don't need to make the yaml files, just choose the right version in the dropdown menu in the last cell

it DID load the YAML file, look:

Loading config from: C:\TEMP\Stable diff\Autom111\stable-diffusion-webui\models\Stable-diffusion\TwoPointZeroDreamTest_step_6000.yaml
LatentDiffusion: Running in v-prediction mode
DiffusionWrapper has 865.91 M params.
Loading weights [1b51225c] from C:\TEMP\Stable diff\Autom111\stable-diffusion-webui\models\Stable-diffusion\TwoPointZeroDreamTest_step_6000.ckpt
Applying xformers cross attention optimization.
Model loaded.

**Comment by LIQUIDMIND111** [comment]
i noticed that training on 2.0, since its 768x768, takes WAY LONGER and MORE STEPS to train..... right?

**Comment by LIQUIDMIND111** [comment]
> The V2 is a bit hard to train, less overfitting though

yep, unfortunately, either 1.5 or 2.0 if training a 768 pixels model, at 100 images, 10000 steps IS NOT enough,,, since those are the recommended steps for 512... so have to try, 20000 steps..... it might take many days on FREE Collab

**Comment by TheLastBen** [comment]
I trained this at 12k https://huggingface.co/TheLastBen/hrrzg-style-768px

**Comment by LIQUIDMIND111** [comment]
> I trained this at 12k https://huggingface.co/TheLastBen/hrrzg-style-768px V1.5 768px

here it says that even at 12000 steps you onlt did 1500 steps of text encoder? how much PERCENT was that?

"""1k steps text encoder at the beginning and 500 steps at the end.""

**Comment by LIQUIDMIND111** [comment]
> yes low textenc because it's a style, not a particular face

is 20% text encoder good for a 25,000 steps experiment?

**Comment by LIQUIDMIND111** [comment]
got any ideas why losses are now so high compared to before??? Average loss is ""0.349""

**Comment by TheLastBen** [comment]
for the A100 the loss looks high
